lib/ansible/cli/doc.py

```diff
@@ -90,13 +90,16 @@ def _load_argspec(self, role_name, collection_path=None, role_path=None):
 
         try:
             with open(path, 'r') as f:
-                return from_yaml(f.read(), file_name=path)
+                data = from_yaml(f.read(), file_name=path)
+                return data.get('argument_spec', {})
```

Comment:
`argument_spec` or `argument_specs`?

Reply:
I went back and forth on it, too. In the end, I decided on the singular since I thought of it as a single "spec" for all of the arguments. But I guess the case could be made for a "spec" for each argument, making plural seem more appropriate. Any preference?

Reply:
It should probably be plural to indicate there can be more than one. Though if you have `argument_specs['main']` most of the time, I can see that looking a bit funny. But I think it would server as a good reminder there can be additional specs.

Reply:
Good argument. Will change it.